### PR TITLE
Insticator Bid adapter: Adds Support for Video

### PR DESCRIPTION
**Pull request by EugeneVigonny**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Added support for linear video

- contact email [eugene_vigonny@lineate.com](mailto:eugene_vigonny@lineate.com)
- [x] official adapter submission

- A link to a PR on the docs repo: https://github.com/prebid/prebid.github.io/pull/3807

### CODE REVIEW

**Comment by lksharma** [comment]
Hi @EugeneVigonny, 
Please also update the documentation repo: https://github.com/prebid/prebid.github.io

**Comment by EugeneVigonny** [comment]
Hi @lksharma,
I have updated the documentation accordingly to the changes in the bid adapter

File: modules/insticatorBidAdapter.js [hunk 1]
```diff
@@ -252,31 +267,76 @@ function validateSizes(sizes) {
...
     return true;
+  }
+
+  const videoSize = [
+    deepAccess(bid, 'mediaTypes.video.w'),
+    deepAccess(bid, 'mediaTypes.video.h'),
+  ];
+
+  if (
+    !validateSize(videoSize)
```
[the remark this answers is not in the document]
Comment (EugeneVigonny) [reply modules/insticatorBidAdapter.js:319]: Added a check for `mimes`

**Review by lksharma — COMMENTED** [review]
LGTM

**Review by lksharma — APPROVED** [review]

**Pull request merged**